src/k8s/pkg/k8sd/features/calico/cleanup.go

```diff
@@ -0,0 +1,88 @@
+package calico
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os/exec"
+	"regexp"
+	"strings"
+
+	"github.com/canonical/k8s/pkg/snap"
+)
+
+func CleanupNetwork(ctx context.Context, snap snap.Snap) error {
+	output, err := exec.CommandContext(ctx, "ip", "-j", "link", "show").Output()
```

Comment:
use `net.Interfaces` instead

Reply:
that way we use golang's library and do not rely on the arguments passed to an external tool